scripts/c_blob.tmpl

```diff
@@ -43,6 +43,7 @@ static const program_target_t flash = {
     {{'0x%08x' % (algo.symbols['EraseChip'] + header_size + entry)}}, // EraseChip
     {{'0x%08x' % (algo.symbols['EraseSector'] + header_size + entry)}}, // EraseSector
     {{'0x%08x' % (algo.symbols['ProgramPage'] + header_size + entry)}}, // ProgramPage
+    0x0,        // Verify
```

Comment:
This should be a symbol like symbols above it, or 0x0 if the symbol does not exist